hw01_hello_otus/go.mod

```
@@ -1,3 +1,5 @@
-module github.com/fixme_my_friend/hw01_hello_otus
+module github.com/zakharovay_otusgo/hw01_hello_otus
```

```suggestion
module github.com/cmekc/zakharovay-otusgo/hw01_hello_otus
```
Следует указать полный путь до модуля, включая имя автора/организации и репозиторий